store/imagestore/store.go

```diff
@@ -767,3 +768,15 @@ func keyToString(k []byte) string {
 	}
 	return fmt.Sprintf("%s%x", hashPrefix, k)[0:lenKey]
 }
+
+func labelsToString(inLabels types.Labels) string {
+	// take a copy to avoid changing the original slice
+	ls := append(types.Labels(nil), inLabels...)
+	labelsort.By(labelsort.RankedName).Sort(ls)
+
+	var out []string
+	for _, l := range ls {
+		out = append(out, fmt.Sprintf("%q:%q", l.Name, l.Value))
+	}
+	return "[" + strings.Join(out, ", ") + "]"
```

Comment:
This doesn't look too nice, and seems to used exactly once in the code for error-printing. Should we get rid of it?